### DISCUSSION TITLE
Build Eclipse RCP application to .exe file

### DISCUSSION THREAD

**Discussion by trongnd32** [JaneWardSandy/eclipse-pde-partial-idea#109] [General]
Hello, 
thank for creating an awesome plugin. My Eclipse RCP application project run perfectly in IntelliJ with this. 

Now I need to build my project to .exe file, that can publish and run on other computers. This action can be done easily on Eclipse (by perform: right click on MyApp.product -> Export... -> Eclipse product). 

Can this plugin do that? Or have any way to do that in IntelliJ? If you want problem be more specific, I will provide more information about my application and my scenario.

Thanks for reading.

Best regards,
Trong

**Comment by JaneWardSandy** [comment]
IDEA does not support these contents. They are all provided by plug-ins for OSGI-related functions.

At present, this plug-in only partially supports PDE. 

I'm very sorry that there are no plans to support the creation/maintenance of RAC/RCP projects in the future.
